tsconfig.base.json

```diff
@@ -31,7 +31,7 @@
       }
     ],
     "paths": {
-      "@payload-config": ["./test/admin/config.ts"],
+      "@payload-config": ["./test/plugin-form-builder/config.ts"],
```

Comment:
Probably should revert this

Reply:
Nah I've accepted modifications to this file long ago

Reply:
Ohh I see haha

Ok there's merge conflicts on it though, otherwise I think this PR is a good idea